pkg/validator/server.go

```diff
@@ -32,7 +32,7 @@ type validatingWebhookServer struct {
 }
 
 // NewValidatingWebhook returns a validatingWebhookServer with the defaultValidators that were previously registered.
-func NewValidatingWebhook(ctx context.Context, webhookConfigName, osmNamespace, osmVersion, meshName string, enableReconciler, validateTrafficTarget bool, certManager *certificate.Manager, kubeClient kubernetes.Interface, policyClient policy.Controller) error {
+func NewValidatingWebhook(ctx context.Context, webhookConfigName, osmNamespace, osmVersion, meshName string, enableReconciler, validateTrafficTarget bool, certManager *certificate.Manager, kubeClient kubernetes.Interface, policyClient k8s.PassthroughInterface) error {
```

Comment:
Just for posterity: this is a valid use of the passthrough interface because the validatingwebhook concept is Kubernetes specific.

Reply:
Could you clarify the significance of the passthrough interface? I thought the interface was for methods that were not specific to kubernetes.

Also, should `policyClient` be updated to a more generic name since the interface has methods for more than just the policy resources?

Reply:
Ya I agree, it should probably either be the raw k8s.Client, or a meshCatalog, or compute.Interface

Reply:
Maybe I'm misunderstanding the passthrough interface then; I thought it was for deliberate abstraction leaking in specific cases. Regardless, I do agree the variable should be renamed

Reply:
Oh, that wasn't my intent. I was trying to convey that I think this is a good, intended use of the passthrough interface because it's fine to let abstractions leak into validatingWebhook logic because validatingwebhook logic is explicitly a Kubernetes concept